src/extract_data/get_data_from_csv.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
```

Comment:
arr ok yep could extact common features just cba the logic differs where to stop and start reading from csv file.